src/components/NavBar.js

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+
+function NavBar({navLinks, style}) {
+
+    const linksToDisplay = Object.entries(navLinks).map(([href, label]) => {
+        return (
+            <li key={href}>
+                <a href={href}>{label}</a>
+            </li>
+        )
+    })
+
+  return (
+    <nav>
+        <ul className={style.nav}>
+            {linksToDisplay}
```

Comment:
Since I'm not familiar with react, out of curiosity is there a way to loop within the "template" return section? Is this a standard to have a variable to loop up here instead of doing something like this:

```suggestion
  return (
    <nav>
        <ul className={style.nav}>
            {Object.entries(navLinks).map(([href, label]) => (
              <li key={href}>
                <a href={href}>{label}</a>
              </li>
            )}
```

Reply:
Yeah, it can be done this way. I think I usually put them in variables to help the readability of the return statement because it can get a bit messy, but in this case beings it's just returning the one thing makes sense to have the template in the return

Reply:
Yeah, this is the case a lot of times and often comes down to personal preference. I'm good either way here. I was more curious than anything else.